src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -61,9 +61,9 @@ Please download %2$s from the website."</string>
     <string name="revanced_hide_general_ads_title">Hide general ads</string>
     <string name="revanced_hide_general_ads_summary">Hides general ads.</string>
     <string name="revanced_hide_music_ads_title">Hide music ads</string>
-    <string name="revanced_hide_music_ads_summary">Hides ads before playing a music.</string>
+    <string name="revanced_hide_music_ads_summary">Hides ads before playing music.</string>
```

Comment:
I assume this also hides ads for podcasts. If true, perhaps it should say one of the following instead of `music`:

- `media`
- `videos`
- `tracks`
- or it can say `music/podcasts`

Reply:
This hides all ads that appear on music, music videos, podcasts and [livestreams](https://www.reddit.com/r/YoutubeMusic/comments/1cyt5zh/i_found_out_that_you_can_actually_listen_to_live/)

`Hide media ads` seem most appropriate